lib/rules/dot-notation.js

```diff
@@ -51,18 +51,30 @@ module.exports = {
                     node.computed &&
                     node.property.type === "Literal" &&
                     validIdentifier.test(node.property.value) &&
-                    (allowKeywords || keywords.indexOf("" + node.property.value) === -1)
+                    (allowKeywords || keywords.indexOf(`${node.property.value}`) === -1)
```

Comment:
Oh, good call. I'll see what other opportunities there might be.

On Aug 26, 2016 10:29 PM, "Toru Nagashima" <notifications@github.com> wrote:

> In lib/rules/dot-notation.js
> <https://github.com/eslint/eslint/pull/6989#discussion_r76509316>:
>
> > @@ -51,18 +51,30 @@ module.exports = {
> >                      node.computed &&
> >                      node.property.type === "Literal" &&
> >                      validIdentifier.test(node.property.value) &&
> > -                    (allowKeywords || keywords.indexOf("" + node.property.value) === -1)
> > +                    (allowKeywords || keywords.indexOf(`${node.property.value}`) === -1)
>
> In this case, *personally*, I prefer String(node.property.value) instead.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eslint/eslint/pull/6989/files/346e1cbdf22426535f3bb747525afaeab3301d3a#r76509316>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARWepeZX8ihdc4Ee_fYLMURIgoWMktTks5qj67-gaJpZM4JulxS>
> .
>